### PR TITLE
Move to stable channels + bump to 4.13

### PR DESCRIPTION
**Pull request by locriandev**
Ref. https://issues.redhat.com/browse/ART-5087

**Is this a BUG FIX or a FEATURE ?**:

/kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by locriandev** [comment]
Test build:
- https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2263082
- https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2263121

File: manifests/art.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 updates:
-  - file: "{MAJOR}.{MINOR}/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml" # relative to this file
+  - file: "stable/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml" # relative to this file
```
Comment (cybertron) [inline manifests/art.yaml:2]: If we're going to move from version-specific directories to stable then the directory needs to be renamed too. It looks like that is not happening in this PR right now.

Comment (locriandev) [reply manifests/art.yaml:2]: Looks to me it is: https://github.com/openshift/kubernetes-nmstate/pull/315/files#diff-705a95d35966128e600ba98080c46c11e9c384b996e43a8e64823df68e18eecd

**Comment by locriandev** [comment]
We have both container and bundle builds:

- [ose-kubernetes-nmstate-operator-container-v4.13.0-202212050852.p0.gf1ec00a.assembly.test](https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2290091)
- [ose-kubernetes-nmstate-operator-bundle-container-v4.13.0.202212050852.p0.gf1ec00a.assembly.test-1](https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2290125)

**Comment by locriandev** [comment]
Closing in favor of https://github.com/openshift/kubernetes-nmstate/pull/353

**Pull request closed without merging**